lib/govuk_index/elasticsearch_presenter.rb

```diff
@@ -63,23 +73,37 @@ def withdrawn?
     end
 
     def expanded_links
-      payload["expanded_links"] || {}
+      @_expanded_links ||= payload["expanded_links"] || {}
+    end
+
+    def expanded_links_item(key)
+      expanded_links[key] || {}
+    end
+
+    def values_from_collection(collection, key)
+      collection.map { |item| item[key] }
     end
 
-    def organisations
-      expanded_links["organisations"] || {}
+    def organisation_slugs(type)
+      expanded_links_item(type).map do |content_item|
+        content_item["base_path"].sub('/government/organisations/', '').sub('/courts-tribunals/', '')
```

Comment:
This might be too much work, but is there any chance that we could push this logic into the publishing API? It seems a bit fragile - rummager shouldn't have to know how to parse URLs for different formats.

Alternatively, could rummager store them with their full paths? The applications which use them would have to search by path rather than by slug.